demos/ensembles/ensemble-link-prediction-example.ipynb

```diff
@@ -321,6 +321,7 @@
       "\n",
```

Comment:
<p>Are we keeping the <code>['...']</code>s  for now?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1008/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
oh I thought we had fixed the stripping of these warnings?

Yeah keeping them for now because it seems like an issue with tensorflow, e.g. tensorflow's official keras train example has these warnings (https://www.tensorflow.org/guide/keras/train_and_evaluate). I saw someone post a work around of SO but it doesn't worth the effort to implement for all the demos when hopefully this will be fixed in a future release.

Reply:
Why have all the leftover `['...']`s appeared now? Those left-over pieces from the warning removal (#826) weren't there before.

> hopefully this will be fixed in a future release

At the moment, this seems more hopeful than not, as it doesn't seem anyone has filed an issue against tensorflow https://github.com/tensorflow/tensorflow/issues?q=is%3Aissue+is%3Aopen+sample_weight+modes+were+coerced+from . Maybe we can do it; it should be ~easy, since the example in their documentation serves as a reproducer.

Reply:
My guess is that these warnings appeared here because this is the first time this notebook has been run with `tensorflow` 2.1.0. 

I'll file an issue with tensorflow and see what they say

Reply:
filed an issue with tensorflow: https://github.com/tensorflow/tensorflow/issues/37500

wow they have a lot of open issues!

Reply:
@huonw @kjun9 I got a response. Turns out the warning has already been fixed and the `tf-nightly` version doesn't have the warning

Reply:
Great. I wonder if that fix is included in the 2.2.0 release that seems like it'll happen soon (https://github.com/tensorflow/tensorflow/releases/tag/v2.2.0-rc0).